os/include/tinyara/pm/pm.h

```diff
@@ -635,6 +635,23 @@ enum pm_state_e pm_querystate(int domain);
  ****************************************************************************/
 void pm_dvfs(int div_lvl);
 
+/****************************************************************************
```

Comment:
If CONFIG_PM is checked for compiling up_set_dvfs(), then it should be checked in the header file too. Otherwise up_set_dvfs gets declared in header file but will never be defined

Reply:
Added as suggested~